File: engine/model/src/main/scala/tools/packet/playerinteraction/GetTradeResult.scala
package tools.packet.playerinteraction

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class GetTradeResult(private var _number: Byte, private var _operation: Byte) extends PacketInput {
  def number: Byte = _number

  def operation: Byte = _operation

  override def opcode(): SendOpcode = SendOpcode.PLAYER_INTERACTION
}

File: engine/engine-model/src/main/java/tools/packet/character/interaction/GetTradeResult.java
package tools.packet.character.interaction;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record GetTradeResult(Byte number, Byte operation) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.PLAYER_INTERACTION;
   }
}